examples/material/main.rs

```diff
@@ -102,8 +102,8 @@ impl SimpleState for Example {
         println!("Put camera");
 
         let mut transform = Transform::default();
-        transform.set_xyz(0.0, 0.0, -12.0);
-        transform.rotate_local(Vector3::y_axis(), std::f32::consts::PI);
+        transform.set_translation_xyz(0.0, 0.0, -12.0);
+        transform.prepend_rotation_y_axis(std::f32::consts::PI);
```

Comment:
should be `append` right? I haven't expanded the code, so if it's got no parent then it's equivalent

Reply:
Whether or not there's a parent doesn't affect whether or not it is equivalent: what affects it is what rotations have already been applied. Since the transform was just created, it has no rotation, so the local ("prepend") axes are equivalent to the parent ("append") axes, and semantically we want to rotate with respect to the parent here anyway based on reading the surrounding code/context

Reply:
Oh yes, I should use my brain to think, instead of simply doing "old name to new name" consistency mapping 😛